### ISSUE TITLE
What to do when exceeding maximum URL length?

### ISSUE THREAD

**Issue by dosse** [sdmx-twg/sdmx-rest#52]
Please provide approach/solution for huge data selections when code selection exceeds maximum URL length.

**Comment by dosse** [comment]
Thanks, but I actually meant if the SDMX standard could provide an alternative way to specifying long query parameters through the URL, so that e.g. bigger (data) selections can be supported. 

One possible alternative approach maybe to first POST a query, store that query on the server with a query ID which is returned to the client and request the query result later with GET by ID.

**Example**

```
POST /query
query=somethingverylong...

201 Created
Location: /query/01543164876

```
then

```
GET /query/01543164876

200 Ok
... your query results here...
```

The ID could be a hash of all the query parameters asked by the client. This allows to reuse IDs since recreating a query will return an existing ID for which the result may (or may not) be already cached.

**Comment by sosna** [comment]
Addressed in the upcoming v1.3.0 release. Additional improvements might be discussed for future releases.
